src/platform/Darwin/dnssd/DnssdImpl.cpp

```diff
@@ -64,26 +61,6 @@ void LogOnFailure(const char * name, DNSServiceErrorType err)
     }
 }
 
-/**
- * @brief Starts a timer to wait for the resolution on the kSRPDot domain to happen.
- *
- * @param[in] timeoutSeconds The timeout in seconds.
- * @param[in] ResolveContext The resolve context.
- */
-CHIP_ERROR StartSRPTimer(uint16_t timeoutInMSecs, ResolveContext * ctx)
-{
-    // Check to see if a user default value exists for the SRP timeout. If it does, override the timeoutInMSecs with user default
-    // value. To override the timeout value, use ` defaults write org.csa-iot.matter.darwin SRPTimeoutInMSecsOverride
-    // <timeoutinMsecs>` See UserDefaults.mm for details.
-    timeoutInMSecs = GetUserDefaultDnssdSRPTimeoutInMSecs().value_or(timeoutInMSecs);
-
-    VerifyOrReturnValue(ctx != nullptr, CHIP_ERROR_INCORRECT_STATE);
-    ChipLogProgress(Discovery, "Starting timer to wait for %d milliseconds for possible SRP resolve results for %s", timeoutInMSecs,
```

Comment:
So we used to start a 0ms timer (as in, do one spin around the event loop before acting) but now we won't?